src/pretix/plugins/statistics/views.py

```diff
@@ -162,4 +163,53 @@ def get_context_data(self, **kwargs):
 
         ctx['has_orders'] = self.request.event.orders.exists()
 
+        ctx['seats'] = {}
+
+        if not self.request.event.has_subevents or (ckey != "all" and subevent):
+            ev = subevent or self.request.event
+            if ev.seating_plan_id is not None:
+                seats_qs = ev.free_seats(sales_channel=None, include_blocked=True)
+                ctx['seats']['purchased_seats'] = len(ev.seats.annotate(has_op=Count('orderposition')))
```

Comment:
This is incorrect, since it might also include canceled orders etc. Maybe we just count all seats and subtract blocked seats and free seats? :)